std/c/linux.zig

```diff
@@ -25,3 +25,5 @@ pub extern "c" fn getauxval(__type: c_ulong) c_ulong;
 
 pub const dl_iterate_phdr_callback = extern fn (info: *dl_phdr_info, size: usize, data: ?*c_void) c_int;
 pub extern "c" fn dl_iterate_phdr(callback: dl_iterate_phdr_callback, data: ?*c_void) c_int;
+
+pub extern "c" fn sigaltstack(ss: ?*stack_t, old_ss: ?*stack_t) c_int;
```

Comment:
these can go in `std/c.zig` yeah? they're the same on every platform

Reply:
They are, but they're not supported by every platform and that's why they've been duplicated.

Reply:
I suppose that's better than a link error if accidentally used. Makes sense to me.